sql/glam_etl/fenix_view_clients_daily_scalar_aggregates_v1/view.sql

```diff
@@ -0,0 +1,11 @@
+-- Manually written view that handles time-partitioning
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.glam_etl.fenix_view_clients_daily_scalar_aggregates_v1`
+AS
+SELECT
+  * EXCEPT (submission_date),
+  DATE(_PARTITIONTIME) AS submission_date
```

Comment:
Would this change the `submission_date` to be the time of insertion into the respective `fenix_clients_daily*` tables?

Reply:
No, the `submission_date` should stay the same because it's defined by `--destination_table=${destination_table}\$${SUBMISSION_DATE//-/}`in the run script e.g. `fenix_clients_daily_scalar_aggregates_baseline_v1$20200219`.